configs/components/ruby_smb.rb

```diff
@@ -0,0 +1,5 @@
+component "ruby_smb" do |pkg, settings, platform|
```

Comment:
Our Vanagon config expects this file has to be called `rubygem-ruby_smb.rb` and the component to be called `ruby_smb`.